kong/plugins/prometheus/exporter.lua

```diff
@@ -489,7 +443,7 @@ local function metric_data(write_fn)
 
   -- notify the function if prometheus plugin is enabled,
   -- so that it can avoid exporting unnecessary metrics if not
-  prometheus:metric_data(write_fn, not is_prometheus_enabled())
+  prometheus:metric_data(write_fn, not IS_PROMETHEUS_ENABLED)
```

Comment:
Is this covered by tests? Are we sure this still works the same way?

Reply:
looks like this test is exercising the code -> https://github.com/Kong/kong/blob/master/spec/03-plugins/26-prometheus/07-optional_fields_spec.lua#L119

Reply:
Correct.